properties.schema

```diff
@@ -172,15 +171,15 @@
                 "default": "",
                 "inputType": "Asset:image",
                 "validators": ["required"],
-                "help": "Image displayed in the popup"
+                "help": "Image to be displayed in a popup when this item is selected."
```

Comment:
Did you mean to add a full stop 🇬🇧 / period 🇺🇸 here? 😉

Reply:
I did, yes. And yes, I know there's no consistency as to whether `help` text should include a full stop or not. Happy to make it consistent if we can agree on which style it should be.

Reply:
Just looking at the rest of the file, it looks like full stops have only been bothered with if there's more than one sentence. Happy to go with that for now and not set a steadfast rule.